README.md

```diff
@@ -1,15 +1,16 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Math-Magicians
```

Comment:
- [ ] **[OPTIONAL]** I would recommend you include steps on how to set up and run the project locally